clients/python/Makefile

```diff
@@ -85,6 +85,7 @@ install-dev: _check_venv_active ## install packages for development
 .PHONY: install-test
 install-test: _check_venv_active install-dev ## install packages for testing client
 	pip install -r $(PYTHON_DIR)/requirements/test.txt
+	ipython kernel install --name "python3" --user
```

Comment:
Q: this is to start a jypyter kernel? why would I need that?

Reply:
I believe the purpose of this one is to create  a jupyter kernel with a specific name. That is one way to ensure I would get the right kernel. But in fact it seems it is not needed. So I just deleted this line.